tableauserverclient/server/server.py

```diff
@@ -95,6 +97,10 @@ def baseurl(self):
         return "{0}/api/{1}".format(self._server_address, str(self.version))
 
     @property
+    def namespace(self):
```

Comment:
Is 'xml_namespace' a better name? (No strong opinion here)

Reply:
I doubt anyone outside of TSC contributors will use that property.  I don't have a strong opinion either way.